app/services/schedule_service.rb

```diff
@@ -8,11 +8,27 @@ def initialize(edition)
   end
 
   def schedule(user: nil, reviewed: false)
-    scheduling = Scheduling.new(pre_scheduled_status: edition.status, reviewed: reviewed)
+    scheduling = Scheduling.new(pre_scheduled_status: edition.status,
+                                reviewed: reviewed)
+
     set_edition_status(scheduling, user)
     create_timeline_entry(scheduling)
 
-    create_publish_intent(edition)
+    create_publish_intent
+    schedule_to_publish
+  end
+
+  def reschedule(user: nil)
+    unless edition.scheduled?
+      raise "Edition must be scheduled in order to reschedule"
+    end
+
+    scheduling = edition.status.details
+    set_edition_status(scheduling, user)
+    create_timeline_entry(scheduling)
```

Comment:
I'd have thought we'd have a different timeline entry for updating it.

Reply:
Sure - it makes sense to me both ways, but the flexibility is good.